Form-Controls/index.html

```diff
@@ -9,19 +9,74 @@
   <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+  <!-- <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <script src="index.js"></script> -->
```

Comment:
Please remove any extraneous javascript! This week is about pure HTML and CSS :)